app/src/main/java/com/example/internetshop/domain/data/repository/ProductsCategoryRepository.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.internetshop.domain.data.repository
+
+import com.example.internetshop.domain.data.model.product.SimpleProduct
+import io.reactivex.Single
+
+interface ProductsCategoryRepository {
+    fun getProductsCategory(categoryName: String): Single<List<SimpleProduct>>
```

Comment:
As for me would be better to rename both repository and his method to `CategoryProducts...`.

Reply:
renamed